src/components/Subject/Subject.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './Subject.scss';
+import CalenderIcon from '../../components/icons/calender.icon.js';
+import UserIcon from '../../components/icons/user.icon';
+import UsersIcon from '../../components/icons/users.icon';
+import File from '../../components/icons/file-text.icon.js';
+import IconText from '../../components/IconText/IconText';
+
+class Subject extends React.Component {
+    render() {
+        const subject = this.props.subject;
```

Comment:
We should unify this: const { subject } = this.props;

Reply:
not done yet